libqfieldsync/layer.py

```diff
@@ -203,6 +247,69 @@ def apply(self):
             != self.geometry_locked_expression
         )
 
+        has_changed |= (
+            bool(self.layer.customProperty("QFieldSync/tracking_session_active"))
+            != self.tracking_session_active
+        )
```

Comment:
We should think for a more optimal way to detect the changes, this is pain. Not for this PR, general comment.